crypter/shield-crypt/shield-crypt.go

```diff
@@ -0,0 +1,82 @@
+package main
```

Comment:
Can you move this into `cmd/shield-crypt` so that it follows the pattern of all the other binaries?